conformance/tests/httproute-invalid-cross-namespace-parent-ref.go

```diff
@@ -39,17 +37,6 @@ var HTTPRouteInvalidCrossNamespaceParentRef = suite.ConformanceTest{
 		gwNN := types.NamespacedName{Name: "same-namespace", Namespace: "gateway-conformance-infra"}
 		routeNN := types.NamespacedName{Name: "invalid-cross-namespace-parent-ref", Namespace: "gateway-conformance-web-backend"}
 
-		// The Route must have an Accepted Condition with a ParentRefNotPermitted Reason.
-		t.Run("HTTPRoute with a cross-namespace ParentRef where no ReferenceGrants allows such a reference, has an Accepted Condition with status False and Reason ParentRefNotPermitted", func(t *testing.T) {
```

Comment:
We should circle back and update this test to check for the `NotAllowedByListeners` reason, but can do that separately.

Reply:
Sounds good, I created https://github.com/kubernetes-sigs/gateway-api/issues/1705 so that we remember to circle back :+1: